tests/api.rs

```diff
@@ -33,14 +32,10 @@ mod tests {
         Client, SubscriptionClient,
     };
 
-    static METRICS_INIT: Once = Once::new();
-
-    // Initialize the metrics system. Idempotent.
+    // Initialize the metrics system.
     fn init_metrics() -> oneshot::Sender<()> {
-        METRICS_INIT.call_once(|| {
-            vector::trace::init(true, true, "info");
-            let _ = vector::metrics::init();
-        });
+        vector::trace::init(true, true, "info");
```

Comment:
Dropped the `sync::Once` wrapper, since tests that use it now run in isolated processes.